.github/workflows/release-main.yml

```
@@ -150,7 +150,11 @@ jobs:
 
       - if: ${{ needs.release-please.outputs.release_created }}
         name: Tar signed bundle
-        run: tar czf konvoy-image-bundle-${{ needs.release-please.outputs.release_tag_name }}_darwin_amd64.tar.gz konvoy-image-bundle_darwin_amd64
+        run: |
+          unzip -o konvoy-image.zip
+          mv konvoy-image konvoy-image-bundle_darwin_amd64
```

Does this need to be like:
```suggestion
          mv konvoy-image konvoy-image-bundle_darwin_amd64/konvoy-image
```

---

mv should replace whatever is there